README.md

```diff
@@ -36,30 +36,26 @@ The example below shows the complexity that is hidden with `google-cloud-kvstore
 
 #### With `@google-cloud/datastore`:
 ```js
-var key = datastore.key(['KeyValue', 'key']);
+const key = datastore.key(['KeyValue', 'key']);
 
 datastore.save({
   key: key,
   value: 'value'
-}, function() {});
+}, () => {});
 
-datastore.get(key, function() {});
+datastore.get(key, () => {});
```

Comment:
It's not really readable :) If it's just a callback on success, can we at least `console.log('done');` inside?

Reply:
One thing at a time :) Coming back around to promisify this stuff.